### PR TITLE
Add package installer property resource filter on status

### PR DESCRIPTION
**Pull request by codeforgreen**
#5469

### CODE REVIEW

File: hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/5431-version_canonicalizer_fails_capabilitystatement_dstu2.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 type: fix
 issue: 5431
-jira: SMILE-5306
+jira: SMILE-7589
```
Comment (jmarchionatto) [inline hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/5431-version_canonicalizer_fails_capabilitystatement_dstu2.yaml:4]: Can't see how this change is related

File: hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            arguments(valueSet0, ourChildConceptCount),
+            arguments(valueSet1, 4 + ourChildConceptCount * 2),
+            arguments(valueSet2, 2 + ourChildConceptCount),
+            arguments(valueSet3, ourChildConceptCount)
+      );
+   }
+
+   @ParameterizedTest
+   @MethodSource(value = "parametersValueSets")
+   public void testExpandValueSet_whenUsingHierarchicalCodeSystem_willExpandSuccessfully(ValueSet theValueSet, int theExpectedConceptExpensionCount) {
```
Comment (jmarchionatto) [inline hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java:100]: typo: Expension

File: hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+public class ValueSetExpansionWithHierarchyR4Test extends BaseTermR4Test {
```
Comment (jmarchionatto) [inline hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java:21]: Great test!

**Review by jmarchionatto — APPROVED** [review]
Great work! Approved with a couple of minor details

File: hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/5431-version_canonicalizer_fails_capabilitystatement_dstu2.yaml (hunk 1 above)
Comment (codeforgreen) [reply hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/5431-version_canonicalizer_fails_capabilitystatement_dstu2.yaml]: The commit in the pull request was wrong, it was from an issue which was already merged into master, I messed up the branch somehow. Sorry about that. I've updated the pull request just now.

File: hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java (hunk 3 above)
Comment (codeforgreen) [reply hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java]: thank you!

File: hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/5431-version_canonicalizer_fails_capabilitystatement_dstu2.yaml (hunk 1 above)
Comment (jmarchionatto) [reply hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/5431-version_canonicalizer_fails_capabilitystatement_dstu2.yaml]: Not a problem

File: hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java (hunk 2 above)
Comment (codeforgreen) [reply hapi-fhir-jpaserver-test-r4/src/test/java/ca/uhn/fhir/jpa/term/ValueSetExpansionWithHierarchyR4Test.java]: I've fixed this typo in this pull request. This change had been already merged.

**Pull request merged**